FWCore/Framework/interface/RunForOutput.h

```diff
@@ -57,6 +58,10 @@ namespace edm {
     Timestamp const& endTime() const { return aux_.endTime(); }
     MergeableRunProductMetadata const* mergeableRunProductMetadata() const { return mergeableRunProductMetadata_; }
 
+    /**\return Reusable index which can be used to separate data for different simultaneous LuminosityBlocks.
```

Comment:
I'm confused, how does `RunIndex` help to differentiate `LuminosityBlock`s?

Reply:
It doesn't, that is a cut-n-paste error :)